pootle/static/css/auth.css

```diff
@@ -63,9 +63,7 @@
     box-sizing: border-box;
     padding: 1em;
     width: 50%;
-
```

Comment:
I think this is the way to go.  But it's quite a bit of work to setup groups and then cleanup and I'm not sure it works well without a lot of maintenance.

I suggest I drop that and switch off that check for now.  I'll play a bit with the properties order and try PR that apart from this.